test/test-hash.4th

```diff
@@ -0,0 +1,11 @@
+\ Copyright 2019 nomennescio
+s" string hash" describe#{
+  s" short strings" it#{
+    <{ s" Hello World!" s# -> s" Hello World!" s# }>
+    <{ s" Hello Worlds!" s# -> s" Hello Worlds!" s# }>
+  }#
+  s" failing compares" it#{
+    <{ s" Hello World!" s# -> s" Hello Worlds!" s# }>
+    <{ s" Hello Worlds!" s# -> s" Hello Worlds! " s# }>
```

Comment:
Can you write example tests that outputs expected and actual values before the assertion?

Reply:
I'm not sure how well this shows up in Codewars, but here you go

Reply:
The next step would be to add custom formatting

Reply:
This will look like this.

![image](https://user-images.githubusercontent.com/639336/62389597-18693700-b515-11e9-9201-3dc46545c566.png)

You can use Kumite to see this. Just go to new Kumite, disable tests and just write to stdout. For example
```javascript
console.log(`

<DESCRIBE::>string hash

<IT::>short strings

<PASSED::>Test Passed

<PASSED::>Test Passed

<COMPLETEDIN::>0.211 ms

<COMPLETEDIN::>0.864 ms
`);
```

Reply:
Moved text